_pages/algorithm-development.md

```diff
@@ -6,6 +6,11 @@ excerpt: "Basic Guides for Algorithm Developers"
 show_related: false
 redirect_from:
   - /algorithm-development/basic-guides/
+menus:
+  devcenter:
```

Comment:
(explanatory comment 2/2)
Then each page you want to appear as an item in a submenu gets this `menus` section in its front matter, in the form:
```yaml
[menu identifier]:
  url: "what url the menu link should have"
  title: "what the menu link should say"
  weight: "where in menu it should appear, starting at 1"
```